.github/workflows/stale.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
       - uses: actions/stale@v7
         with:
-          stale-pr-message: 'This pull request has gone a while without any activity. Tagging the Trino developer relations team: @bitsondatadev @colebow @mosabua'
+          stale-pr-message: 'This pull request has gone a while without any activity. Tagging the Trino developer relations team: @colebow @mosabua'
```

Comment:
in fact i think we should use a team handle instead of pinging peoples directly.

cc @losipiuk